nightwatch_tests/admin_page.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  tags: ["admin"],
+
+  "1. Check that when i press Manager, click here, I see 'return to mood entry' page ": async function (
+    browser
+  ) {
+    browser.url("https://project32.s3.eu-west-2.amazonaws.com/index.html");
+    const cssSelector = "#root > div > div > button";
+    await browser.click("#root > div > div > button");
```

Comment:
Ditto from `./smoke_tests`. 
```suggestion
    await browser.click(cssSelector);
```